oak-it/src/test/java/org/apache/jackrabbit/oak/composite/CompositeNodeStoreTest.java

```diff
@@ -46,12 +46,10 @@
 
 import javax.sql.DataSource;
 
-import com.google.common.base.Predicate;
-import com.google.common.base.Predicates;
-import com.google.common.collect.ImmutableSet;
-import com.google.common.collect.Iterables;
-import com.google.common.collect.Lists;
 import org.apache.commons.io.FileUtils;
+import org.apache.jackrabbit.guava.common.base.Predicate;
+import org.apache.jackrabbit.guava.common.base.Predicates;
+import org.apache.jackrabbit.guava.common.collect.Iterables;
```

Comment:
same here... if the imports were on the same lines as before it would be easier to review (and see that some imports didn't get adjusted.